### ISSUE TITLE
Add default color methods to C4Color

### ISSUE THREAD

**Issue by C4Framework** [C4Framework/C4iOS#238]
Add methods like:

```
.blueColor()
```

**Comment by C4Framework** [comment]
Added methods like `.blue()` instead of `.blueColor()`, because `C4Color.blueColor()` is a bit redundant.

Have a look at: https://github.com/C4Framework/C4iOS/blob/238-Add-Color-Methods/C4Core/C4Color.swift#126-192

**Comment by aleph7** [comment]
Look at my changes. It kind of pollutes the namespace but you always do `C4Core.red`

**Comment by C4Framework** [comment]
Yeah, I was thinking that having them as properties so you don't have to do () would be nice, simpler.

This is a nice alternative. But, I'm not sure about referencing it from Core... That's a bit unintuitive, but not overly... If there were other "global" variables that had the same style then this would be a strong approach.

However, with this approach people can also get away with:

```
obj.fillColor = red
```

Do you think things like the following would also work:

```
public let PI = M_PI
```

or 

```
public let C4Logo = C4Image("logo")
```

**Comment by aleph7** [comment]
You don't have to reference it from Core, you can also just do `red`, which is nice. And yes, we should add more constants.

**Comment by C4Framework** [comment]
awesome. this is nice.

there were a bunch of nice constants before, like C4RED etc.

**Comment by aleph7** [comment]
Fixed with e3ed468d82a5d99442987358ba48a547c23cb8f5
